tools/engine_tool/lib/src/logger.dart

```diff
@@ -77,7 +77,8 @@ class Logger {
     runZoned<void>(() {
       try {
         sink.write(message);
-      } catch (_) { // ignore: avoid_catches_without_on_clauses
+      } catch (_) {
+        // ignore: avoid_catches_without_on_clauses
```

Comment:
I thought these ignore lines had to be on the same line. If the analyzer is fine with this, then this ignore line might not even be needed.

Reply:
This file was auto formatted and doesn't have any analyzer issues aside from TODOs.

I've tried removing the ignore line.